plugins/woocommerce/includes/class-wc-brands.php

```diff
@@ -276,7 +276,7 @@ public static function init_taxonomy() {
 						'add_new_item'      => __( 'Add New Brand', 'woocommerce' ),
 						'new_item_name'     => __( 'New Brand Name', 'woocommerce' ),
 						'not_found'         => __( 'No Brands Found', 'woocommerce' ),
-						'no_terms'          => __( 'No brands', 'woocommerce' ),
+						'no_terms'          => __( 'No Brands', 'woocommerce' ),
```

Comment:
```suggestion
						'no_terms'          => __( 'No brands', 'woocommerce' ),
```

I think this should be lowercase to match tag and category.

Reply:
This has been my first commit, but CodeRabbit AI noticed that `Brand` has the first letter capitalized in other strings. I don't have a strong opinion - what do you think?

Reply:
<img width="1537" height="850" alt="image" src="https://github.com/user-attachments/assets/485bd80b-7543-4471-ae0a-39596413df93" />

I think it's better to be consistent with other taxonomies, like tags and categories.

Reply:
I updated it! 💪